src/test/java/org/opensrp/web/rest/ClientFormResourceTest.java

```diff
@@ -316,55 +315,53 @@ public void testSearchForFormByFormVersionShouldReturnNextJsonValidatorVersion()
         assertEquals("0.0.3", jsonNode.get("clientFormMetadata").get("version").textValue());
         assertTrue(jsonNode.get("clientFormMetadata").get("isJsonValidator").booleanValue());
     }
-
+    
     @Test
     public void testAddClientFormWhenGivenJSON() throws Exception {
         String formIdentifier = "opd/reg.json";
-        String formVersion = "0.1.1";
```

Comment:
This were removed since my understanding was, we aren't making the form version optional but removing the ability for a user/dev to actually set one.  In that case we don't need to pass in a form version.